documentation/modules/exploit/multi/http/apache_activemq_upload_jsp.md

```diff
@@ -0,0 +1,119 @@
+## Vulnerable Application
+
+This module exploits a vulnerability in Apache ActiveMQ 5.x before 5.14.0 which
+allows remote attackers to upload and execute arbitrary files via an HTTP PUT
```

Comment:
Again execute them as which user? The user that installed Apache ActiveMQ? The user that runs a specific service? `root`? `apache`? Its hard to tell the risk without this information.

Reply:
Added a note:
> You should get a shell under the process running ActiveMQ.